source/_integrations/tuya.markdown

```diff
@@ -25,79 +24,123 @@ ha_platforms:
 ha_dhcp: true
 ---
 
-The `tuya` integration is the main integration to integrate [Tuya Smart](https://www.tuya.com) related platforms, except the Zigbee and Bluetooth(BLE) hub. This includes devices linked with the Tuya, Smart Life, and Jinvoo Smart apps. You will need your account information (username, password and account country code) from one of these apps to discover and control devices which related to your account.
+The Tuya Home Assistant integration is developed for controlling **Powered by Tuya (PBT)** devices, which is officially maintained by the Tuya Developer Team.
 
-**Important**: Not all Tuya devices are supported by the `tuya API` used by this integration. For more details refer to [TuyaHA Library](https://github.com/PaulAnnekov/tuyaha).
+You can find more information for this integration on [Tuya Integration Github](https://github.com/tuya/tuya-home-assistant?_source=418498f8d2819b5b1654a0c12c5325cd).
 
-There is currently support for the following device types within Home Assistant:
-
-- **Climate** - The platform supports the air conditioner and heater.
-- **Cover** - The platform supports curtains.
-- **Fan** - The platform supports most kinds of Tuya fans.
-- **Light** - The platform supports most kinds of Tuya light.
-- **Scene** - The device state in frontend panel will not change immediately after you activate a scene.
-- **Switch** - The platform supports switch and socket.
+It applies to smart devices that you have added to the Tuya Smart or Smart Life app.
 
 {% include integrations/config_flow.md %}
 
-During configuration, be careful to select the [country code](https://www.countrycode.org/) and the platform corresponding to those used by you in the app. Once configuration flow is completed, the devices configured in your app will be automatically discovered. Additionally, if your username or e-mail address isn't accepted, please try using your phone number (minus the country code) as your username.
+## Supported device categories
+- **Climate**: Supports air conditioner, heater, thermostat.
+- **Cover**: Supports curtain, curtain switch.
+- **Fan**: Supports fan, air purifier, humidifier, dehumidifier.
+- **Light**: Supports light, light strip, ambient light, light string, humidifier's light, Ceiling light.
+- **Scene**: Supports smart scene.
+- **Switch**: Supports switch, socket, power strip, smart kettle, breaker, pet water feeder, air purifier, diffuser.
+
+## Quick installation and startup
+
+### Configure on the Tuya IoT Platform
+
+#### Prerequisites
```

Comment:
I think we don't need these chapters. Instead, make a prerequisites chapter to instruct on how to set up the Tuya IoT Platform, next include our config flow instructions.